backend/wayland/wl_seat.c

```diff
@@ -49,7 +54,7 @@ static void pointer_handle_motion(void *data, struct wl_pointer *wl_pointer,
 	struct wlr_wl_pointer *wlr_wl_pointer =
 		(struct wlr_wl_pointer *)dev->pointer;
 	if (!wlr_wl_pointer->current_output) {
-		wlr_log(L_ERROR, "pointer motion event without current output");
+		wlr_log(L_DEBUG, "pointer motion event without current output");
```

Comment:
You don't have to do it now, but I'd like to see all the pointer motion stuff have `L_TRACE`.

Reply:
I'd really rather not wait for `L_TRACE` to come in, as there is not even a PR open with it. If that's the only thing left I can either revert this or keep as is but let's please merge this?

Reply:
Yeah it's fine. Sometimes I just make these notes for other readers or myself.